### PR TITLE
Feature/convert string to pascalcase

### LINKED ISSUE #4096: Write a Kotlin program to convert string to pascalcase
### Description

Write a Kotlin program to convert string to pascalcase

Pascalcase is a style of writing in which the first letter of each word is capitalized.

```
Input  : "hello world"
Output : "HelloWorld"
```
<details>
<summary>How to contribute</summary>

- Save the solution in `program/ConvertStringToPascalcase.kt` file
- Add `ConvertStringToPascalcase.kt` file in `convert-string-to-pascalcase` folder

</details>

### PR DESCRIPTION
**Pull request by rksp25**
### Why:

Closes codinasion/codinasion#3046

### What's being changed:

Add pascalcase in Kotlin

### CODE REVIEW

File: program/convert-string-to-pascalcase/ConvertStringToPascalcase.kt [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import java.util.Scanner
+
+fun main() {
+    print("Enter String : ")
+    println()
+    var min = readLine()
+
+    println("Pasacl Case: " + ConvertStringToPascalcase(min!!.split(" ")))
+
```
Comment (anandfresh) [inline program/convert-string-to-pascalcase/ConvertStringToPascalcase.kt:9]: 'Pascal' , typo error

File: program/convert-string-to-pascalcase/ConvertStringToPascalcase.kt [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+private fun ConvertStringToPascalcase(args: List<String>): String {
+    val buffer = StringBuffer()
+    for((index,day) in args.withIndex()){
+        if(index == 0) {
+            buffer.append(day.get(0).uppercase()).append(day.subSequence(1,day.length))
+        } else {
+            buffer.append(" ").append(day.get(0).uppercase()).append(day.subSequence(1,day.length))
+        }
+    }
```
Comment (anandfresh) [inline program/convert-string-to-pascalcase/ConvertStringToPascalcase.kt:20]: Getting error while executing this program 

```
OpenJDK 64-Bit Server VM warning: Options -Xverify:none and -noverify were deprecated in JDK 13 and will likely be removed in a future release.
prog.kt:16:38: error: unresolved reference: uppercaseChar
            buffer.append(day.get(0).uppercaseChar()).append(day.subSequence(1,day.length))
                                     ^
prog.kt:18:50: error: unresolved reference: uppercaseChar
            buffer.append(" ").append(day.get(0).uppercaseChar()).append(day.subSe
```

**Review by anandfresh — CHANGES_REQUESTED** [review]

File: program/convert-string-to-pascalcase/ConvertStringToPascalcase.kt (hunk 2 above)
Comment (harshraj8843) [reply program/convert-string-to-pascalcase/ConvertStringToPascalcase.kt:20]: work fine on https://www.tutorialspoint.com/online_kotlin_compiler.php

**Review by harshraj8843 — APPROVED** [review]
Thanks for contributing 👍🏻

**Pull request merged**